ui/app/pages/routes/index.js

```diff
@@ -141,7 +141,7 @@ class Routes extends Component {
         <Authenticated path={CONFIRM_ADD_TOKEN_ROUTE} component={ConfirmAddTokenPage} exact />
         <Authenticated path={CONFIRM_ADD_SUGGESTED_TOKEN_ROUTE} component={ConfirmAddSuggestedTokenPage} exact />
         <Authenticated path={NEW_ACCOUNT_ROUTE} component={CreateAccountPage} />
-        <Authenticated path={`${CONNECT_ROUTE}/:id`} component={PermissionsConnect} exact />
+        <Authenticated path={`${CONNECT_ROUTE}/:id?`} component={PermissionsConnect} />
```

Comment:
Why was the `?` added? It doesn't look like we're ever expecting this route to lack an id.
```suggestion
        <Authenticated path={`${CONNECT_ROUTE}/:id`} component={PermissionsConnect} />
```